src/fsharp/FSharp.Core/seq.fs

```diff
@@ -842,6 +797,486 @@ namespace Microsoft.FSharp.Collections
     [<RequireQualifiedAccess>]
     [<CompilationRepresentation(CompilationRepresentationFlags.ModuleSuffix)>]
     module Seq =
+//        type ISeqEnumerable<'T> =
+//            abstract member Fold<'State> : folder:('State->'T->'State) -> state:'State -> 'State
+
+        module SeqComposer =
```

Comment:
rec module - shiny new feature I haven't used :-)

Personally I don't mind the 'and' syntax, but I'll give it a burl!